gum/backend-darwin/gumcodesegment-darwin.c

```diff
@@ -101,6 +101,9 @@ struct _GumCsRequirements
 static gboolean gum_code_segment_try_realize (GumCodeSegment * self);
 static gboolean gum_code_segment_try_map (GumCodeSegment * self,
     gsize source_offset, gsize source_size, gpointer target_address);
+static gboolean gum_code_segment_try_remap (GumCodeSegment * self, gsize source_offset,
+    gsize source_size, gpointer target_address);
+static gboolean gum_code_segment_is_realize_supported (void);
```

Comment:
This forward-declaration of `gum_code_segment_is_realize_supported()` should go before `gum_code_segment_try_realize()` as the implementation is before it.